install/updateVersion.sh

```diff
@@ -132,22 +144,16 @@ function merge_files() {
   cat $SRC/istio-ingress.yaml.tmpl >> $ISTIO
   cat $SRC/istio-egress.yaml.tmpl >> $ISTIO
 
-  cp ${SRC}/istio-initializer.yaml.tmpl $ISTIO_INITIALIZER
-
-  cp $ISTIO $ISTIO_CLUSTER_WIDE
-  sed -i=.bak "s/# authPolicy: MUTUAL_TLS/authPolicy: MUTUAL_TLS/" $ISTIO_CLUSTER_WIDE
-#TODO the CA templates can be combined
-  cat $SRC/istio-cluster-ca.yaml.tmpl >> $ISTIO_CLUSTER_WIDE
 
-  # Deploy istio-ca always, required to enable MTLS for mixer and pilot
-  cat $SRC/istio-namespace-ca.yaml.tmpl >> $ISTIO
+  cp $ISTIO $ISTIO_ONE_NAMESPACE
+  # restrict pilot controllers to a single namespace in the test file
+  sed -i=.bak "s|args: \[\"discovery\", \"-v\", \"2\"|args: \[\"discovery\", \"-v\", \"2\", \"-a\", \"${ISTIO_NAMESPACE}\"|" $ISTIO_ONE_NAMESPACE
```

Comment:
it'd be nice to support in shared env 1 namespace for the app and 1 namespace for istio; but I guess we can change that later